README.md

```diff
@@ -9,7 +9,7 @@
 ## Objectives
 
 -   Contrast the defintions of "property", "attribute", and "method"
--   From within a method, reference properties of the same object using `this`.
+-   From within a method, access properties of the same object using `this`.
```

Comment:
This does not have to do with the commit message and therefore should be in a different commit.